hooks/post_gen_project.py

```diff
@@ -1,9 +1,217 @@
+
+# remove the template file to avoid it getting added to the new repo
+os.remove('tfmodule-template.tfvars')
+
+YES_OPTIONS, NO_OPTIONS = frozenset(["y", "yes"]), frozenset(["n", "no", ""])
+PERM_OPTIONS = frozenset(["push", "pull", "admin"])
+
+
+def multiple_choices(question):
+    selection = []
+    additional = "Would you like to add another (y/n)? [n]: "
+
+    check = True
+    while check:
+        choice = input(question).lower()
+        selection.append(choice)
+        check = check_choice(additional)
+    return selection
+
+
+def check_choice(question):
+    while True:
+        choice = input(question).lower()
+        if choice in YES_OPTIONS:
+            return True
+        if choice in NO_OPTIONS:
+            return False
+
+
+def get_collaborators():
+    # Questions that have multiple layers to them
+    collaborators = []
+    collab_q = "Grant an organizational team access to the repo(y/n)? [n]: "
+
+    if check_choice(collab_q):
+        add_team = True
+
+        while add_team:
+            teams_q = "What team would you like to grant access? "
+            team = input(teams_q).lower()
+
+            if team != "":
+                perm = None
+                while perm not in PERM_OPTIONS:
+                    perms_q = (
+                        "What permissions would you like to grant them"
+                        " (push, pull, admin)? ")
+                    perm = input(perms_q).lower()
+                collaborators.append({"name": team, "permission": perm})
+
+            another_q = "Would you like to add another team (y/n)? [n]: "
+            add_team = check_choice(another_q)
+
+    return collaborators
+
+
+def run_terraform(directory, target_module):
+    terraform = Terraform(directory)
+    terraform.init(from_module=target_module)
+    ret_code, _, _ = terraform.apply(auto_approve=True, capture_output=False)
+
+    return ret_code
+
+
+def open_pr(source_repo, working_dir):
+    """Opens a pull request."""
+
+    # clone the repo
+    git.Repo.clone_from(
+        source_repo,
+        "tempdir"
+    )
+
+    # Move the cloned contents up a directory
+    # because we can't clone to an existing directory
+    for file_name in os.listdir("tempdir"):
+        shutil.move(
+            join(working_dir, "tempdir", file_name),
+            join(working_dir, file_name)
+        )
+    os.rmdir("tempdir")
+
+    # init the repo obj
+    repo = git.Repo(working_dir)
+
+    # create a new branch
+    new_branch = repo.create_head('init')
+
+    # checkout the new branch
+    new_branch.checkout()
+
+    # add all files in working dir to a new commit in the newly created branch
+    repo.git.add('--all')
+    repo.git.commit(m='Module initialization')
+
+    # push the changes
+    repo.git.push('--set-upstream', 'origin', new_branch)
+
+    # open a PR
+    subprocess.check_call(['hub', 'pull-request', '--no-edit'])
+
+
+if __name__ == "__main__":
+
+    print("\nPost hook questions\n")
```

Comment:
Questions are only asked if they create an organizational repo so the print statement should be moved down under elif (cookiecutter.create_org_repo) portion